recipes/_configure.rb

```diff
@@ -22,6 +22,7 @@
   end
 
   include_recipe 'cerner_splunk::_configure_server'
+  include_recipe 'cerner_splunk::_configure_logs'
```

Comment:
I don't think we want log-local.cfg to be created if the `node['splunk']['logs']` hash is empty or is not set. So maybe something like 
```
include_recipe 'cerner_splunk::_configure_logs' unless node['splunk']['logs'].empty?
```
Also, you will need to default this attribute to an empty hash in the attributes. https://github.com/cerner/cerner_splunk/blob/stable/attributes/_configure.rb.

Reply:
I don't think there's any harm in having an empty log-local.cfg (splunk just falls back to the regular log.cfg for anything not defined in log-local), but I can add the empty check if we want.

Ah, thanks for the tip about the attributes.

Reply:
Addressed in these two commits:
https://github.com/cerner/cerner_splunk/pull/191/commits/9067447497130d498df2944c87233eec4e7fd019
https://github.com/cerner/cerner_splunk/pull/191/commits/129f676ebf640bdb4e4a69dc3c8ca2caa34c6eb9

Reply:
> I don't think there's any harm in having an empty log-local.cfg

I don't think it hurts to have an empty log-local.cfg either, but I think the problem here is that the next time when someone runs chef-client with this change, it will end up creating the `log-local.cfg` file (empty or not), causing Splunk to restart (because we are touching the file marker). So even if someone has not configured these logging attributes, this change will end up forcing a restart, which is not the expected behavior.